lms/djangoapps/user_tours/models.py

```diff
@@ -0,0 +1,27 @@
+""" Models for the User Tour Experience. """
+
+from django.contrib.auth import get_user_model
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+User = get_user_model()
+
+
+class UserTour(models.Model):
```

Comment:
Might want to add an admin.py for this model too

Reply:
I had a personal question about that. Because it exists 1-1 with users, I wasn't sure if that was too many rows that it causes issues in the admin. I don't have access to the user table in prod so couldn't remember if it was enabled at all or not.

Reply:
Nah, you're right. It won't hurt to have the detail view and I'll look into if the standard user list view exists. Can just mirror that for this table. Enrollment is the disabled one because that is x times bigger than user, so maybe user is fine.

Reply:
I think you are right though that User tables might also be guarded somehow? I don't want to try to load it in prod, in case I affect the site though.  :)  But might be worth a look at its admin class.

Reply:
Added admin table. Didn't see any special restrictions. I think we were thinking enrollments.